src/50_Samples_%26_Templates/Live_Blog.html

```diff
@@ -172,12 +185,20 @@
       <p id="summary">This is a sample article demonstrating how to write a live blog in AMP. It demonstrates the usage of amp-live-list component which allows to create live blogs.</p>
     </div>
     <!-- #### Blog Posts -->
-    <!-- Use amp-live-list to implement a live blog. Find an amp-live-list example [here](/components/amp-live-list) -->
+    <!-- Use amp-live-list to implement a live blog. Find an amp-live-list example [here](/components/amp-live-list). Use `<div pagination>` tag to include pagination, find more [here](https://www.ampproject.org/docs/reference/components/amp-live-list) -->
```

Comment:
sug: "Use `<div pagination>` tag to include pagination, find more [here](https://www.ampproject.org/docs/reference/components/amp-live-list)" -> "You can implement pagination for live-blogs using [`<div pagination>`](https://www.ampproject.org/docs/reference/components/amp-live-list)